### PR TITLE
tar, wc, zsh: update French translation

### PR DESCRIPTION
**Pull request by mcombeau**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/tar.md [hunk 1]
```diff
@@ -6,25 +6,29 @@
 
 - Crée une archive à partir de fichiers :
 
-`tar cf {{cible.tar}} {{fichier1 fichier2 fichier3}}`
+`tar cf {{cible.tar}} {{fichier1}} {{fichier2}} {{fichier3}}`
```
Comment (EmilySeville7cfg) [inline pages.fr/common/tar.md:9]:
```suggestion
`tar cf {{chemin/vers/cible.tar}} {{chemin/vers/fichier1 chemin/vers/fichier2 ...}}`
```

Mandatory fix ([original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/tar.md) must be updated): path format is [wrong](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths), `tar` doesn't mandate exactly 3 files to be passed.

File: pages.fr/common/tar.md [hunk 2]
```diff
@@ -6,25 +6,29 @@
...
 
-- Crée une archive gzip :
+- Crée une archive gzip à partir de fichiers :
 
-`tar czf {{cible.tar.gz}} {{fichier1 fichier2 fichier3}}`
+`tar czf {{cible.tar.gz}} {{fichier1}} {{fichier2}} {{fichier3}}`
```
Comment (EmilySeville7cfg) [inline pages.fr/common/tar.md:13]:
```suggestion
`tar czf {{chemin/vers/cible.tar.gz}} {{chemin/vers/fichier1 chemin/vers/fichier2 ...}}`
```

The same issue as above.

File: pages.fr/common/tar.md [hunk 3]
```diff
@@ -6,25 +6,29 @@
...
+- Crée une archive gzip à partir d'un répertoire en utilisant son chemin relatif :
 
-`tar xf {{source.tar[.gz|.bz2|.xz]}}`
+`tar czf {{target.tar.gz}} --directory={{chemin/vers/répertoire}} .`
 
-- Extrait une archive dans un dossier cible :
+- Extrait une archive (compressée) dans le dossier courant en affichant la liste des fichiers traités :
 
-`tar xf {{source.tar}} -C {{dossier}}`
+`tar xvf {{source.tar[.gz|.bz2|.xz]}}`
```
Comment (EmilySeville7cfg) [inline pages.fr/common/tar.md:21]:
```suggestion
`tar xvf {{chemin/vers/source.tar[.gz|.bz2|.xz]}}`
```

Mandatory fix ([original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/tar.md) must be updated): the same as above.

```suggestion
`tar xvf {{chemin/vers/source.tar.gz|chemin/vers/source.tar.bz2|chemin/vers/source.tar.xz}}`
```

Optional fix ([original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/tar.md) can be updated): "nested" extension list can be unwrapped to placeholder with alternatives.

File: pages.fr/common/tar.md [hunk 4]
```diff
@@ -6,25 +6,29 @@
...
+- Crée une archive gzip à partir de fichiers :
 
-`tar czf {{cible.tar.gz}} {{fichier1 fichier2 fichier3}}`
+`tar czf {{cible.tar.gz}} {{fichier1}} {{fichier2}} {{fichier3}}`
 
-- Extrait une archive (compressée) dans le dossier courant :
+- Crée une archive gzip à partir d'un répertoire en utilisant son chemin relatif :
 
-`tar xf {{source.tar[.gz|.bz2|.xz]}}`
+`tar czf {{target.tar.gz}} --directory={{chemin/vers/répertoire}} .`
```
Comment (EmilySeville7cfg) [inline pages.fr/common/tar.md:17]:
```suggestion
`tar czf {{chemin/vers/cible.tar.gz}} --directory={{chemin/vers/répertoire}} .`
```

The same issue as above.

File: pages.fr/common/tar.md [hunk 5]
```diff
@@ -6,25 +6,29 @@
...
 
-- Extrait une archive dans un dossier cible :
+- Extrait une archive (compressée) dans le dossier courant en affichant la liste des fichiers traités :
 
-`tar xf {{source.tar}} -C {{dossier}}`
+`tar xvf {{source.tar[.gz|.bz2|.xz]}}`
+
+- Extrait une archive (compressée) dans un répertoire cible :
+
+`tar xf {{source.tar[.gz|.bz2|.xz]}} --directory={{répertoire}}`
```
Comment (EmilySeville7cfg) [inline pages.fr/common/tar.md:25]:
```suggestion
`tar xf {{chemin/vers/source.tar[.gz|.bz2|.xz]}} --directory={{répertoire}}`
```

```suggestion
`tar xf {{chemin/vers/source.tar.gz|chemin/vers/source.tar.bz2|chemin/vers/source.tar.xz}} --directory={{répertoire}}`
```
The same issue as above.

File: pages.fr/common/tar.md [hunk 6]
```diff
@@ -6,25 +6,29 @@
...
 
 - Crée une archive compressée, en utilisant le suffixe de l'archive pour déterminer le programme de compression :
 
-`tar caf {{cible.tar.xz}} {{fichier1 fichier2 fichier3}}`
+`tar caf {{cible.tar.xz}} {{fichier1}} {{fichier2}} {{fichier3}}`
```
Comment (EmilySeville7cfg) [inline pages.fr/common/tar.md:29]:
```suggestion
`tar caf {{chemin/vers/cible.tar.xz}} {{chemin/vers/fichier1 chemin/vers/fichier2 ...}}`
```

The same issue as above.

File: pages.fr/common/tar.md [hunk 7]
```diff
@@ -6,25 +6,29 @@
...
 
-- Liste le contenu d'une archive tar :
+- Liste les fichiers contenus dans une archive tar :
 
 `tar tvf {{source.tar}}`
```
Comment (EmilySeville7cfg) [inline pages.fr/common/tar.md:33]:
```suggestion
`tar tvf {{chemin/vers/source.tar}}`
```

The same issue as above.

File: pages.fr/common/zsh.md [hunk 8]
```diff
@@ -14,12 +14,16 @@
...
-`zsh --xtrace {{path/to/script.zsh}}`
+`zsh --xtrace {{chemin/vers/script.zsh}}`
 
 - Démarre une session shell interactive en mode verbeux, qui affiche chaque commande avant de l'exécuter :
 
 `zsh --verbose`
+
+- Exécute une commande spécifique dans `zsh` sans motifs génériques d'expansion des noms de fichier :
+
+`noglob {{command}}`
```
Comment (EmilySeville7cfg) [inline pages.fr/common/zsh.md:29]:
```suggestion
`noglob "{{command}}"`
```

Optional fix ([original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/zsh.md) can be updated): add missing quotes.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- Changes were requested as some mandatory fixes found.
- Thanks for a contribution! ❤️

File: pages.fr/common/tar.md (hunk 1 above)
Comment (mcombeau) [reply pages.fr/common/tar.md:9]: Ah, thank you for clarifying this point, I will keep it in mind for my future contributions.

**Comment by mcombeau** [comment]
Thank you for your careful review, @EmilySeville7cfg ! I have made the necessary changes and will keep all of this in mind for future contributions.

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**